lib/trellis/utils/output.py

```diff
@@ -55,7 +55,7 @@ def replace_item_with_key(obj, result):
     )
 
     if should_replace:
-        if 'key' in result._result[item]:
```

Comment:
This PR exposed a bug in this code since `result._result[item]` was a string containing `"key"` (in `public_keys`) and Python's `in` operator is flexible for dict keys or substrings.